onboarding/src/Components/UsersList/UserListRow.js

```diff
@@ -2,18 +2,23 @@ import React from "react";
 import { Link } from 'react-router-dom';
 
 function UserListRow(props) {
+    let avatar = "/onboarding/static/images/unknown-profile.jpg";
+
+    if(props.user.avatar && props.user.avatar.length > 1)
+        avatar = props.user.avatar;
+
     return(
         <div className="card mb-2">
             <div className="card-body">
                 <div className="row align-items-center">
 
                     <div className="col-auto">
-                        <Link to="/employee_profile" className="user-avatar user-avatar-xl"><img src="/onboarding/static/images/unknown-profile.jpg" alt="" /> <span className="avatar-badge idle" title="idle"></span></Link>
+                        <Link to={{ pathname: "/employee_profile", state: { user: props.user } }} className="user-avatar user-avatar-xl"><img src={ avatar } alt="" /> <span className="avatar-badge idle" title="idle"></span></Link>
                     </div>
 
                     <div className="col">
                         <h3 className="card-title">
-                            <Link to="/employee_profile">{ props.user.name }</Link> <small className="text-muted"><a href="#">{ props.user.email }</a></small>
+                            <Link to={{ pathname: "/employee_profile", state: { user: props.user } }}>{ props.user.name }</Link> <small className="text-muted"><Link to={{ pathname: "/employee_profile", state: { user: props.user } }}>{ props.user.email }</Link></small>
                         </h3>
```

Comment:
I think here also should be link for example `pathname: "/employee_profile/" + employee_id`.

Reply:
Could be :-)
But employee/user details come with `state: { user: props.user }`
We will see what to do with that next.